server/testdb/environments.go

```diff
@@ -113,7 +108,12 @@ func (td *postgresDB) GetEnvironments(ctx context.Context, take, skip int32, que
 		sql += condition
 	}
 
-	sql = sortQuery(sql, sortBy, sortDirection, environmentSortingFields)
+	sortFields := map[string]string{
```

Comment:
just for consistency, should we name this `sortingFields` instead of `sortFields`?

Reply:
given it's just an internal, temporary variable, it's not very relevant, but I'll fix it